content/dei/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Diversity, Equity, and Inclusion"
+date: 2021-09-17
+draft: false
+---
+
+Healthy communities are built when everyone's voice is heard,
+when their perspective is valued, and when their work is recognized.
+Better technical and social solutions can be found through wide participation.
+We are therefore dedicated to expanding our community in a welcoming and inclusive way,
+with particular focus on increased participation by people who identify with groups that are  underrepresented in STEM.
```

Comment:
Do you want to say STEM or scientific open source? I know this is a nit but I think it's better to be explicit about our focus than be too general.

Reply:
I think this was meant to say that we'd like to see more participation from people underrepresented in STEM, since we rely on those fields to provide contributors.  The Scientific Open Source community is tiny, so it is likely not to be well represented along various axes.

More importantly, however, I think the idea is for you and Hannah to own this page and edit it to your heart's content.

Reply:
@melissawm Maybe "underrepresented in STEM and the open source scientific Python community" or "underrepresented in STEM and the open source scientific Python community, in particular".  I don't have a strong opinion and am happy to use whatever you prefer.

Reply:
I tried to resolve this in 8533240.